tools/publishSamples.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash
+# Script to sync the GitHub web fundamentals samples contents onto GitHub Pages.
+# Run from the location of a clone of the fundamentals repo.
+
+WEBDIR="$(pwd)"
+GHDIR="/tmp/WF-gh-pages"
+
+# - are we in the web dir?
+if [ ! -f "Gruntfile.js" ]; then
+  echo "Please make sure you're in the git checkout location."
+  exit
+fi
+
+echo "----- Getting latest sources from GitHub"
+
+# Git pull
+git checkout master
+git pull https://github.com/Google/WebFundamentals master
+LASTCOMMIT=$(git rev-list HEAD --max-count=1)
+
+if [ $? -ne 0 ]; then
```

Comment:
Any chance of a comment explaining this does? To a shell noob like me, that is nonsense.